tools/cp-caps/README.md

```diff
@@ -221,3 +222,19 @@ RX ver:2015,Data,seq,dst[addr:short,pan:id],src[addr:short,pan:id],sec:no,ie[csl
 TX ver:2015,Data,noseq,dst[addr:short,pan:id],src[addr:short,pan:id],sec:no,ie:no,plen:0 ----------- OK
 RX ver:2015,Data,noseq,dst[addr:short,pan:id],src[addr:short,pan:id],sec:no,ie:no,plen:0 ----------- OK
 ```
+
+### Test mTxInfo field of the radio frame.
+
+The option `-T` or `--tx-info` tests whether the RCP supports the mTxInfo field of the radio frame.
+
+```bash
+$ DUT_ADB_USB=1269UCKFZTAM95OR REF_CLI_SERIAL=/dev/ttyACM0 python3 ./tools/cp-caps/rcp_caps_test.py -T
+mIsSecurityProcessed=True -------------------------------- OK
+mIsSecurityProcessed=False ------------------------------- OK
+mTxDelayBaseTime=now,mTxDelay=500000 --------------------- OK
+mRxChannelAfterTxDone ------------------------------------ OK
+mCsmaCaEnabled = 0 --------------------------------------- OK
+mCsmaCaEnabled = 1 --------------------------------------- OK
```

Comment:
nit: unify the format regarding whether to have spaces on both sides of the equal sign.